### PR TITLE
Skills - Implement Fire Support skill tree

### PR DESCRIPTION
**Pull request by veteran29**
Depends on #61 - Review after it's merged to have proper diff

### CODE REVIEW

File: game/functions/systems/skill/client/actives/fire_support/fn_skill_actives_fireSupport_grenadesCase.sqf [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+    Returns:
+        Nothing
+
+    Example(s):
+        [] call vgm_c_fnc_skill_passives_fireSupport_grenadesCase
+ */
+
+["Fire Support/Grenades Case skill activated"] call vgm_g_fnc_logInfo;
+
+["skill_active_sixthSense", {
```
Comment (Spoffy) [inline game/functions/systems/skill/client/actives/fire_support/fn_skill_actives_fireSupport_grenadesCase.sqf]: Think this runLater name needs an update? :)

File: game/functions/systems/skill/client/actives/fire_support/fn_skill_actives_fireSupport_overwhelmingFire.sqf [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+    Returns:
+        Nothing
+
+    Example(s):
+        [] call vgm_c_fnc_skill_passives_fireSupport_overwhelmingFire
+ */
+
+["Fire Support/Overwhelming Fire skill activated"] call vgm_g_fnc_logInfo;
+
+["skill_active_sixthSense", {
```
Comment (Spoffy) [inline game/functions/systems/skill/client/actives/fire_support/fn_skill_actives_fireSupport_overwhelmingFire.sqf]: Same here r.e. runLater name :)

File: game/mission/stringtable.xml [hunk 3]
```diff
@@ -230,5 +233,39 @@
...
+        <Key ID="STR_VGM_SKILLS_SKILL_FIRE_SUPPORT_LOADOUT_EXPLOSIVES">
+            <Original>Explosives Equipment</Original>
+        </Key>
+        <Key ID="STR_VGM_SKILLS_SKILL_FIRE_SUPPORT_LOADOUT_GRENADIER">
+            <Original>Grenadier and AT Equipment</Original>
+        </Key>
+        <Key ID="STR_VGM_SKILLS_SKILL_FIRE_SUPPORT_LEARN_THE_RHYHTM">
+            <Original>Learn the Rhythm</Original>
+        </Key>
+        <Key ID="STR_VGM_SKILLS_SKILL_FIRE_SUPPORT_LEARN_THE_RHYHTM_DESC">
```
Comment (Spoffy) [inline game/mission/stringtable.xml]: Typo in these string names, should be RHYTHM P)

File: game/configs/mission/cfg_skills.hpp [hunk 4]
```diff
@@ -185,4 +183,90 @@ class vgm_skillTrees {
...
+                class loadout_grenadier: vgm_skillTemplate {
+                    displayName = "$STR_VGM_SKILLS_SKILL_FIRE_SUPPORT_LOADOUT_GRENADIER";
+                    description = "$STR_VGM_SKILLS_SKILL_FIRE_SUPPORT_LOADOUT_GRENADIER_DESC";
+                };
+            };
+
+            class tier_3 {
+                class learnTheRhythm: vgm_skillTemplate {
+                    displayName = "$STR_VGM_SKILLS_SKILL_FIRE_SUPPORT_LEARN_THE_RHYHTM";
+                    description = "$STR_VGM_SKILLS_SKILL_FIRE_SUPPORT_LEARN_THE_RHYHTM_DESC";
```
Comment (Spoffy) [inline game/configs/mission/cfg_skills.hpp]: Typo in RHYTHM here :)

**Review by Spoffy — CHANGES_REQUESTED** [review]

File: game/mission/stringtable.xml (hunk 3 above)
Comment (veteran29) [reply game/mission/stringtable.xml]: Damn this word, I keep typoing it all the time. In Polish it's just `rytm`.

Comment (Spoffy) [reply game/mission/stringtable.xml]: Hah, I like the simplicity of the polish word :D

**Review by Spoffy — APPROVED** [review]

**Pull request merged**